tasks.php

```diff
@@ -586,16 +586,13 @@ function create_task_from_form_submission($formsub)
         case 'notify_new':
             $userSettings =& Settings::get_Settings($pguser);
             $userSettings->add_value('taskctr_notice', 'notify_new');
-            metarefresh(0, "$tasks_url");
-//            TaskHeader("All Open Tasks");
-  //          list_all_open_tasks();
+            metarefresh(0, $tasks_url);
```

Comment:
This is minor, but worth pointing out. If you are on a task page (eg: https://www.pgdp.org/~cpeel/c/tasks.php?action=show&task_id=2229) and use the Receive New Task Alerts button you get shunted to the main tasks page instead of back to the task you were viewing. Ideally this would go back to the task page. One probably-easy solution would be to only show the button if the user was looking at the full task list rather than just a single task.

Either way this isn't super-important so feel free to push back on this request.

Reply:
I've made the button only appear if the user is looking at all tasks. There are some cases where the button does not appear when it could: e.g. after editing a task. It would need more work to fix that ...